cypress/integration/student_demonstrations_growth.js

```diff
@@ -37,24 +37,25 @@ describe.skip('Student competency growth calculation test', () => {
                 };
 
                 cy.visit('/cbl/dashboards/demonstrations/student').then(() => {
-                    cy.withExt().then(({extQuerySelector}) => {
+                    cy.withExt().then((res) => {
                         const studentUsernames = Object.keys(growthCalculationsByStudent);
                         studentUsernames.forEach(studentUsername => {
                             const studentContentAreas = Object.keys(growthCalculationsByStudent[studentUsername]);
                             studentContentAreas.forEach(studentContentArea => {
                                 cy.visit(`/cbl/dashboards/demonstrations/student#${studentUsername}/${studentContentArea}`);
+                                cy.request(`/cbl/dashboards/demonstrations/student#${studentUsername}/${studentContentArea}`).then((res) => {
+                                  expect(res.status).to.eq(200)
+                                })
```

Comment:
why did we add this `cy.request` call? You would normally use `cy.request` to make API calls. In this case we appear to be loading the same page we just navigated the browser to a second time via XHR. The `cy.visit` call would already have failed if the browser was unable to navigate to the page.

I would remove this `cy.request` call -- was there any particular reason you added it?

Reply:
I added `cy.request` because i saw that 
`cy.wait('@studentCompetencyData')
                                  .its('status')
                                  .should('eq', 200)
                                  .then(() => {`
                                  
  wasnt returning the correct status. When I looked at the docs one of the first things that came up was showing how to check that `status` of the `req` object which I thought we were doing. So I figured I would use `cy.request(...)  and then use the `response` object to check the status.